source/modules/configuration/authentication/ldap.rst

```diff
@@ -58,6 +58,8 @@ LDAP directory
 
 * **Default server**: If you have multiple LDAP servers set up, you can only define one default server. Choosing this setting will remove it from the server it was previously set to.
 * **Server** and **Port**: Represent the address and port of the LDAP directory.
+   GLPI can connect to a LDAP directory through a LDAPS connection. To enable this, prefix your server entry by *ldaps://* and change the port to the LDAPS one of your LDAP directory (default 636).
```

Comment:
```suggestion
   GLPI can connect to an LDAP directory through a LDAPS connection. To enable this, prefix your server field with *ldaps://* and change the port to the LDAPS one of your LDAP directory (default 636).
```
Minor English-related changes.
'an' should be used before a word that _sounds_ like it starts with a vowel (sounds like `el·dap`.
Prefix with sounds more natural in this case than prefix by too.